rector.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Core\ValueObject\PhpVersion;
+use Rector\Php74\Rector\LNumber\AddLiteralSeparatorToNumberRector;
+use Rector\Php74\Rector\Property\RestoreDefaultNullToNullableTypePropertyRector;
+use Rector\Set\ValueObject\SetList;
+
+return static function (RectorConfig $rectorConfig): void {
```

Comment:
@yurabakhtin Can we integrate this in a Yii Console Command? Maybe somehow with optional parameter module path?

Reply:
@luke- Done in commit https://github.com/humhub/humhub/pull/6048/commits/2d6d6b38a8c0c427bbbb77040307646a9a308acd:

- `php yii rector/core`
- `php yii rector/module <moduleId>`

Reply:
@yurabakhtin Cool!

Reply:
@luke- If we have the console commands then no need to keep config `rector.php` in the root, so I moved it into the folder `/protected/config` - https://github.com/humhub/humhub/pull/6048/commits/25bf093dccd6cdfc41b49940c827763db6ba9fcf